docs/guides/verify-shapella-upgrade-manual.md

```diff
@@ -0,0 +1,530 @@
+VALIDATOR_DELAYED_TIMEOUT_IN_SLOTS=7200  # 1 day
+VALIDATOR_DELINQUENT_TIMEOUT_IN_SLOTS=28800  # 4 days
+NODE_OPERATOR_NETWORK_PENETRATION_THRESHOLD_BP=100  # 1% network penetration for a single NO
+
+# Time period of historical observations used for prediction of the rewards amount 
+PREDICTION_DURATION_IN_SLOTS=50400  # 7 days
+
+# Max period of delay for requests finalization in case of bunker due to negative rebase
+# twice min governance response time - 3 days voting duration
+FINALIZATION_MAX_NEGATIVE_REBASE_EPOCH_SHIFT=1350  # 6 days
+```
+
+## OracleReportSanityChecker
+
+```python
+# Sanity limit on the number of deposits: not more than ~half of the current DSM deposits capacity (43200 it is)
+# https://github.com/lidofinance/lido-dao/blob/feature/shapella-upgrade/contracts/0.8.9/sanity_checks/OracleReportSanityChecker.sol#L221-L232
+churnValidatorsPerDayLimit = 20000
+
+# Taken from current oracle limit https://etherscan.io/address/0x442af784A788A5bd6F42A01Ebe9F287a871243fb#readProxyContract#F8
+# See https://github.com/lidofinance/lido-dao/blob/feature/shapella-upgrade/contracts/0.8.9/sanity_checks/OracleReportSanityChecker.sol#L47-L50
+oneOffCLBalanceDecreaseBPLimit = 500  # 5%
+
+# See https://research.lido.fi/t/increasing-max-apr-sanity-check-for-oracle-lido-report/3205
+# Related to Consensus Layer rewards only
+annualBalanceIncreaseBPLimit = 1000  # 10%
+
+# See https://github.com/lidofinance/lido-dao/blob/feature/shapella-upgrade/contracts/0.8.9/sanity_checks/OracleReportSanityChecker.sol#L647-L672
+simulatedShareRateDeviationBPLimit = 50  # 0.5%
+
+# Same as the current churn limit in Ethereum (8 validators per epoch)
+# used in ValidatorsExitBusOracle, which has reporting period 8 hours
+maxValidatorExitRequestsPerReport = 600
+
+# See https://github.com/lidofinance/lido-dao/blob/feature/shapella-upgrade/contracts/0.8.9/sanity_checks/OracleReportSanityChecker.sol#L65-L67
+maxAccountingExtraDataListItemsCount = 2
+
+# See https://github.com/lidofinance/lido-dao/blob/feature/shapella-upgrade/contracts/0.8.9/sanity_checks/OracleReportSanityChecker.sol#L69-L71
+# and https://github.com/lidofinance/lido-dao/blob/e45c4d6fb8120fd29426b8d969c19d8a798ca974/contracts/0.8.9/oracle/AccountingOracle.sol#L302-L306
+maxNodeOperatorsPerExtraDataItemCount = 100
+
+# See https://github.com/lidofinance/lido-dao/blob/feature/shapella-upgrade/contracts/0.8.9/sanity_checks/OracleReportSanityChecker.sol#L73-L75
+# and https://research.lido.fi/t/withdrawals-for-lido-on-ethereum-bunker-mode-design-and-implementation/3890
+requestTimestampMargin = 7680  # 2 hours rounded to epoch length
+
+# 27% yearly, in 1e9 so that it multiplied on 365 (link to code)
+# see https://research.lido.fi/t/increasing-max-apr-sanity-check-for-oracle-lido-report/3205
+# and https://github.com/lidofinance/lido-dao/blob/feature/shapella-upgrade/contracts/0.8.9/sanity_checks/OracleReportSanityChecker.sol#L77-L79
+maxPositiveTokenRebase = 750000
```

Comment:
Maybe add a note that the value might be re-visited once the stETH liquidity sources landscape changed